pkg/kv/dynamodb/store.go

```diff
@@ -97,6 +98,11 @@ func (d *Driver) Open(ctx context.Context, kvParams kvparams.Config) (kv.Store,
 				},
 			}))
 	}
+	cfg = cfg.WithHTTPClient(&http.Client{
+		Transport: &http.Transport{
+			MaxConnsPerHost: 10,
```

Comment:
1. Define constant for `10` to explain to the linter this is the max conn per host.
2. Can you check if sdk v1 supports default client that we can only override max connections? the new sdk for example provides a function to get access to the aws default client.

Reply:
1. Done
2. I reduced the scope to the best we can